packages/remirror__styles/all.css

```diff
@@ -594,14 +594,6 @@ button:active .remirror-menu-pane-shortcut,
   position: relative;
 }
 
-.remirror-editor.ProseMirror[contenteditable='false'] {
-  white-space: normal;
-}
-
-.remirror-editor.ProseMirror[contenteditable='true'] {
-  white-space: pre-wrap;
-}
```

Comment:
Possible issue here (though unlikely?) is that our browser support is `since 2017` ([these browsers](https://en.wikipedia.org/wiki/List_of_web_browsers#Historical)).

`white-space: break-spaces` isn't supported by [until more recent versions](https://caniuse.com/mdn-css_properties_white-space_break-spaces).

Perhaps we should just [copy ProseMirror here](https://github.com/ProseMirror/prosemirror-view/blob/aaa50d592074c8063fc2ef77907ab6d0373822fb/style/prosemirror.css#L7-L8)?

```css
.remirror-editor.ProseMirror {
  white-space: pre-wrap;
  white-space: break-spaces;
}
```

Reply:
We already have these two rules, right above the changed lines in this file.

Reply:
Whoops! 🤦‍♂️